Jenkinsfile

```diff
@@ -485,7 +484,13 @@ void TestInVM(deviceMode, deploymentMode, distro, distroVersion, kubernetesVersi
                            loggers=; \
                            atexit () { set -x; kill \$loggers; kill \$( ps --no-header -o %p ); }; \
                            trap atexit EXIT; \
+                           echo CLUSTER=\$CLUSTER TEST_LOCAL_REGISTRY=\$TEST_LOCAL_REGISTRY TEST_DISTRO=\$TEST_DISTRO TEST_DISTRO_VERSION=\$TEST_DISTRO_VERSION TEST_KUBERNETES_VERSION=\$TEST_KUBERNETES_VERSION >_work/new-cluster-config && \
```

Comment:
@pohly Can't we reuse the cluster in case of LCOAL_REGISTRY changes? Is it because it needs a change in docker config(insecure registry)?

Reply:
Correct.